src/common/utils/network-utils.ts

```diff
@@ -0,0 +1,7 @@
+export function getApiUrl(chain: string): string | undefined {
```

Comment:
we should use api urls from network context or constants as this will introduce another source of truth

Reply:
Have to use constants as this code is being run on the server